transaction-status/Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = { workspace = true }
 Inflector = { workspace = true }
 base64 = { workspace = true }
 bincode = { workspace = true }
-borsh = { workspace = true }
+borsh0-9 = { package = "borsh", version = "0.9.3" }
```

Comment:
nit: Can you add a comment to use the workspace-level version once `spl-associated-token-account` uses borsh 0.10?